recipes/koalas/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "koalas" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9ed16834e255924ddfe9a254cbb258b35dd167b727f223bee316ee436778794d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: true  # [py<35]
+
+requirements:
+  host:
+    - numpy >=1.14
+    - pandas >=0.23
+    - pip
+    - pyarrow >=0.10
+    - python
+  run:
+    - numpy >=1.14
+    - pandas >=0.23
+    - pyarrow >=0.10
+    - python
+
+test:
+  requires:
+    - pyspark >=2.4
```

Comment:
If this is needed here should it be a `run` dependency instead?

Reply:
Actually we need the package, but we don't want to install the package automatically since users might have their own installation in their cluster and it might conflict. In that case, can we still move it to `run` dependency?

Reply:
It depends how people are likely to have installed `pyspark` on their cluster. If conda is aware of it (i.e. it comes from a different conda channel or `pip`) then it will be fine and, if not, they can always forcibly remove conda-forge's installation afterwards.

You can check by running `conda list` and looking for `pyspark`.

Reply:
Usually users have their own Spark cluster in advance, which includes `pyspark` libraries. So we don't want to include as a dependency when doing `conda install koalas`. This is the same way when users do `pip install koalas` which does not install `pyspark` as a dependency.

Reply:
@chrisburr Are there anything else I can do for this to be merged?

Reply:
Yes, ping me as I has lost track 😆

LGTM 👍